src/orquestra/sdk/_base/cli/_dorq/_ui/_prompts.py

```diff
@@ -86,10 +89,16 @@ def choice(
         if len(choices) == 1:
             return self._handle_single_option(message, choices[0])
 
+        # We may need to be able to modify the list of choices to add an 'all' option,
+        # so cast it to a list instance.
+        _choices = list(choices)
+        if allow_all:
+            _choices += ["all"]
+
```

Comment:
**thought**: I like this addition, it's a nice utility. However, we don't have unit tests for this part. I wonder if we're approaching a time when we should add test suite against inquirer API mocks.

Reply:
Good idea, do we have a ticket floating around in the backlog for that?

Reply:
Can't see any suitable!

Reply:
There is now! https://zapatacomputing.atlassian.net/browse/ORQSDK-890?atlOrigin=eyJpIjoiMGYwZWFiZGZhNGQ2NGM5Nzg5ZWY4NWQ0OGFhZTU5MmQiLCJwIjoiaiJ9